webpack/assets/javascripts/react_app/components/users/JwtTokens/JwtTokens.js

```diff
@@ -0,0 +1,64 @@
+import React, { Fragment } from 'react';
+import { Button } from '@patternfly/react-core';
+import { KeyIcon } from '@patternfly/react-icons';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { openConfirmModal } from '../../ConfirmModal';
+import { APIActions } from '../../../redux/API';
+import { translate as __ } from '../../../common/I18n';
+
+const JwtTokens = ({ userId }) => {
```

Comment:
What if the `userID` is not set?

Reply:
If the user is logged in, we always have the @user object set and we are sending it from [here](https://github.com/theforeman/foreman/pull/10405/commits/a25111626f9da7e74dd250e6c0706679a7f2828e#diff-270d03ca5d3dfc156ff0a306b43cb88f2157375cfabf49f0c6601657d0090cb4R3) so I don't think that would happen